### PR TITLE
Add ESP

### PR DESCRIPTION
**Pull request by agucova**
The ESP8266 and ESP32 have become highly popular, and the go-to devices for a good chunk of DIY IoT projects, largely replacing the Raspberry Pi in small embedded uses.

**https://github.com/agucova/awesome-esp#readme**

The list includes cool projects, useful libraries, tools and firmware for both microcontrollers. Note that I disabled two linter rules, both, I believe, false positives. (I'll raise an issue in awesome-lint).

**By submitting this pull request I confirm I've read and complied with the [requirements](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) 🖖**

Pull requests reviewed: #1688 and #1687.

### CODE REVIEW

**Comment by agucova** [comment]
unicorn

File: readme.md [hunk 1]
```diff
@@ -146,6 +146,8 @@
 - [Adafruit IO](https://github.com/adafruit/awesome-adafruitio#readme) - Visualize and store data from any device.
 - [Cloudflare](https://github.com/irazasyed/awesome-cloudflare#readme) - CDN, DNS, DDoS protection, and security for your site.
 - [Actions on Google](https://github.com/ravirupareliya/awesome-actions-on-google#readme) - Developer platform for Google Assistant.
+- [ESP](https://github.com/agucova/awesome-esp#readme) - Two low-cost microcontrollers with WiFi, having broad IoT applications.
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
- [ESP](https://github.com/agucova/awesome-esp#readme) - Low-cost microcontrollers with WiFi and broad IoT applications.
```

?

I would drop "two" as there might be more ESP controllers in the future.

**Comment by sindresorhus** [comment]
This should be dropped:

> This is a curated list of awesome projects and code made with both ESP's according to the Awesome Manifesto.

---

In `Both the ESP8266 and ESP32`, linkify the names to their websites, like done with the images.

**Comment by agucova** [comment]
I've dropped it and added the links to the quoteblock.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1223484444629061632

**Pull request merged**